dataplane/linux/bpf_ep_mgr.go

```diff
@@ -500,8 +500,11 @@ func (m *bpfEndpointManager) ensureQdisc(ifaceName string) {
 func (m *bpfEndpointManager) attachWorkloadProgram(endpoint *proto.WorkloadEndpoint, polDirection PolDirection) error {
 	ap := m.calculateTCAttachPoint(tc.EpTypeWorkload, polDirection, endpoint.Name)
 	// Host side of the veth is always configured as 169.254.1.1.
-	hostIP := net.IPv4(169, 254, 1, 1)
-	err := tc.AttachProgram(ap, hostIP)
+	ap.IP = net.IPv4(169, 254, 1, 1)
+	// vxlanMTU is the MTU of workload devices, we need to use another 50 less
```

Comment:
I think this comment is a bit misleading.  The VXLANMTU configuration parameter is the MTU that felix sets on the VXLAN device that it creates for VXLAN IP pools.  That's its original purpose.

So, I think we need to carefully explain where the `-50` come from here.  I think it's something like:

* VXLAN MTU should be the eth0 MTU -50, in order to allow space for VXLAN.
* We also expect that to be the MTU used on veths.
* We do encap on the veths, and there's a bogus kernel MTU check in the BPF helper for resizing the packet, so we have to reduce the apparent MTU by another 50 bytes.

Reply:
Maybe the variable should be renamed ICMP_TOO_BIG_MTU, or something?

Reply:
applied the comment kept the var name